path_login.go

```diff
@@ -137,6 +142,17 @@ func (b *kubeAuthBackend) pathLogin(ctx context.Context, req *logical.Request, d
 	}, nil
 }
 
+func (b *kubeAuthBackend) getAliasName(role *roleStorageEntry, serviceAccount *serviceAccount) (string, error) {
+	switch role.AliasNameSource {
+	case aliasNameSourceSAToken, aliasNameSourceUnset:
+		return serviceAccount.uid(), nil
+	case aliasNameSourceSAPath:
+		return fmt.Sprintf("%s/%s", serviceAccount.Namespace, serviceAccount.Name), nil
+	default:
+		return "", fmt.Errorf("unknown alias_name_source %q", role.AliasNameSource)
+	}
+}
+
 // aliasLookahead returns the alias object with the SA UID from the JWT
 // Claims.
 func (b *kubeAuthBackend) aliasLookahead(_ context.Context, _ *logical.Request, data *framework.FieldData) (*logical.Response, error) {
```

Comment:
We'll also have to update `aliasLookahead` to return the right alias format/value depending on the role's `alias_name_source` value as discussed offline.

Reply:
Yup, working on that now.

Reply:
Support added in https://github.com/hashicorp/vault-plugin-auth-kubernetes/pull/110/commits/40ecafed3570dcfd269833c7834264757cbed356